index.php

```diff
@@ -0,0 +1,54 @@
+<head>
+<link rel="stylesheet" type="text/css" href="normalize.css">
+<link rel="stylesheet" type="text/css" href="style.css">
+<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
+<title>My Collection</title>
+</head>
+<body>
+    <h1>My Handbag Collection</h1>
+    <nav>
+        <div class="navbar">
+            <a class="underline">Collection</a>
+            <a class="underline">Add Item</a>
+            <a class="underline">Edit Item</a>
+        </div>
+    </nav>
+<main>
+</main>
+</body>
+<?php
+$db = new PDO('mysql:host=db; dbname=MyCollection','root','password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query=$db->prepare("SELECT* FROM `bags`;");
```

Comment:
Syntax is SELECT * FROM `bags`;
You are missing a space. This probably doesn't matter for functionality (as MySQL probably deals with it in interpretation) but I'd still amend it.

Also, recall that it is bad practice to use 'SELECT *' at all, as it is possible that your database structure will change during development. I would recommend that you SELECT only the values you want.